### ISSUE TITLE
Improve statistics

### ISSUE THREAD

**Issue by arnecls** [arnecls/raidplaner#12]
- Make the attendance statistics from the settings section public
- Add global DKP statistics next to it

[Original issue on Google Code](https://code.google.com/p/ppx-raidplaner/issues/detail?id=87)

**Comment by LeonGeb** [comment]
You said that the statistics are only visible to the raidlead. Can this be changed so it can be viewed by all who are logged in? It's not that important but it could push the motivation to join raids (if not static anyway).

**Comment by arnecls** [comment]
Yes that's possible (and not very complicated), but IMHO this has to be made an option as some guilds (like my old one) did not want to have this public to avoid finger pointing.

**Comment by LeonGeb** [comment]
Hi Arne, Thx that would be great. I know what you mean but the guild this change is for is ok with it.
